projects/python/simulation/agricultural-simulation-dataset-generator/agricultural-environment-robotti-mavicpro/controllers/robotti/robotti.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020-2023 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from controller import Robot, Camera, Lidar
+
+MAX_SPEED = 6.28
```

Comment:
This parameter can be also moved there.

Reply:
Okay, will collect the parameters in one header file.